source/_patterns/components/00-alerts/00-alert.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="alert {{ alertClass }}" role="alert">
+  <span class="alert-icon pficon {{ alertIcon }}"></span>
+  <div class="alert-text">
```

Comment:
Yes, if it's a new class name we are creating, then it should be BEM and prefixed with `pf-`